src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -402,19 +402,20 @@ static ucs_status_t uct_rc_mlx5_iface_preinit(uct_rc_mlx5_iface_common_t *iface,
     iface->tm.mp.num_strides = 1;
```

Comment:
if we don't support `1` anymore, we should remove it (I guess)

Reply:
no, we use it as an indicator whether MP XRQ is supported or not